api-reference/v1.0/resources/groups_overview.md

```diff
@@ -0,0 +1,125 @@
+    "id": "4c5ee71b-e6a5-4343-9e2c-4244bc7e0938",
+    "deletedDateTime": null,
+    "classification": "MBI",
+    "createdDateTime": "2016-08-23T14:46:56Z",
+    "description": "This is an Outlook group",
+    "displayName": "OutlookGroup101",
+    "groupTypes": [
+        "Unified"
+    ],
+    "mail": "outlookgroup101@service.microsoft.com",
+    "mailEnabled": true,
+    "mailNickname": "outlookgroup101",
+    "preferredLanguage": null,
+    "proxyAddresses": [
+        "smtp:outlookgroup101@microsoft.onmicrosoft.com",
+        "SMTP:outlookgroup101@service.microsoft.com"
+    ],
+    "securityEnabled": false,
+    "theme": null,
+    "visibility": "Public"
+}
+```
+Learn more about Office 365 groups and the administrator experiences [here](https://support.office.com/en-us/article/Learn-about-Office-365-groups-b565caa1-5c40-40ef-9915-60fdb2d97fa2).
+
+## Security groups and mail-enabled security groups
+Security groups are for controlling user access to resources. By checking whether or not a user is a member of a security group, your app can make authorization decisions when that user is trying to access some secure resources in your app. Security groups can have users and other security groups as members.
+
+Mail-enabled security groups are used in the same way as security groups are, but with the added feature of a shared mailbox for the groups. Mail-enabled security groups can't be created through the API, but other group operations will still work here. Learn more in the [Manage mail-enabled security groups Exchange article](https://technet.microsoft.com/en-us/library/bb123521(v=exchg.160).aspx).
+
+### Example of security group
+
+```http
+{
+    "@odata.type": "#microsoft.graph.group",
+    "id": "f87faa71-57a8-4c14-91f0-517f54645106",
+    "deletedDateTime": null,
+    "classification": null,
+    "createdDateTime": "2016-07-20T09:21:23Z",
+    "description": "This group is a Security Group",
+    "displayName": "SecurityGroup101",
+    "groupTypes": [],
+    "mail": null,
+    "mailEnabled": false,
+    "mailNickname": "mysg",
+    "preferredLanguage": null,
+    "proxyAddresses": [],
+    "securityEnabled": true
+}
+```
+## Dynamic membership 
+All types of groups can have dynamic membership rules which automatically add or remove members from the group based on user properties. For example, a "Marketing employees" group would include every user with the department property set to "Marketing", so that new marketing employees are automatically added to the group and employees who leave the department are automatically removed from the group. This rule can be specified in a "membershipRule" field during group creation as ```"membershipRule": 'user.department -eq "Marketing"'``` GroupType must also include ```"DynamicMembership"```. The following request creates a new Office 365 group for the marketing employees: 
+
+```http
+POST https://graph.microsoft.com/v1.0/groups
```

Comment:
Dynamic groups is still in /beta only.  Peter Ciszewski <Peter.Ciszewski@microsoft.com> owns this.